pom.xml

```diff
@@ -38,34 +39,36 @@
   </licenses>
 
   <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
I put the project encoding and JDK version up at the top so they're easy to see.